tests/lib/rules/indent.js

```diff
@@ -0,0 +1,230 @@
+            "        case \"a\":\n" +
+            "            return \"A\";\n" +
+            "        case \"b\":\n" +
+            "            return \"B\";\n" +
+            "    }\n" +
+            "}\n" +
+            "foo();"
+        },
+        {
+            code: "if (a){}",
+            args: [2, "incorrect values for configuration will be ignored"]
+        }
+    ],
+    invalid: [
+        // todo: this test case is works fine in jscs from 1.10.0, fails in 1.9.0
+        /*        {
+         code: "if (array.some(function(){\n" +
+         "  return true;\n" +
+         "})) {\n" +
+         "a++; // ->\n" +
+         "  b++;\n" +
+         "    c++; // <-\n" +
+         "}\n",
+         args: [2, {size: 2}],
+         errors: expectedErrors([[4, 2], [6, 2]])
+         },*/
+
+        {
+            code: "if (a){\n\tb=c;\n\t\tc=d;\ne=f;\n}",
+            args: [2, "tab"],
+            errors: expectedErrors([[3, 1], [4, 1]])
+        },
+        {
+            code: "if (a){\n    b=c;\n      c=d;\n e=f;\n}",
+            args: [2, 4],
+            errors: expectedErrors([[3, 4], [4, 4]])
+        },
+        {
+            code: fixture,
+            args: [2, 2],
+            errors: expectedErrors([
+                [5, 2],
+                [10, 4],
+                [11, 2],
+                [15, 4],
+                [16, 2],
+                [23, 2],
+                [29, 2],
+                [30, 4],
+                [36, 4],
+                [38, 2],
+                [39, 4],
+                [40, 2],
+                [46, 0],
+                [54, 2],
+                [114, 4],
+                [120, 4],
+                [124, 4],
+                [127, 4],
+                [134, 4],
+                [139, 2],
+                [145, 2],
+                [149, 2],
+                [152, 2],
+                [159, 2],
+                [168, 4],
+                [176, 4],
+                [184, 4],
+                [186, 4],
+                [200, 2],
+                [202, 2],
+                [214, 2],
+                [218, 6],
+                [220, 6],
+                [330, 6],
+                [331, 6],
+                [371, 2],
+                [372, 2],
+                [375, 2],
+                [376, 2],
+                [379, 2],
+                [380, 2],
+                // [386, 1],
+                // [388, 1],
+                [399, 2],
+                [401, 2],
+                [405, 4],
+                [407, 4],
+                [414, 2],
+                [416, 2],
+                [421, 2],
+                [423, 2],
+                [440, 2],
+                [441, 2],
+                [447, 2],
+                [448, 2],
+                [454, 2],
+                [455, 2],
+                [461, 6],
+                [462, 6],
+                [467, 6],
+                [472, 6],
+                [486, 2],
+                [488, 2],
+                [534, 6],
+                [541, 6]
```

Comment:
@lo1tuma usually it is not a "good" idea to add lines at top of fixture files, for any tests